packages/frontend/README.md

```diff
@@ -1,36 +1,7 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Metrom frontend
```

Comment:
Can we also add the header image here?

Reply:
Fixed in c0dc1e341febba4af6cce425f572e802f8a851c8.